pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cohere"
-version = "4.2.1"
+version = "4.2.2"
```

Comment:
I think 4.3 is appropriate for a new endpoint

Reply:
updated